src/lib/comms/include/sol-mavlink.h

```diff
@@ -70,7 +70,7 @@ enum sol_mavlink_mode {
      * allowing roll, pitch, and yaw to be controlled normally.
      */
     SOL_MAVLINK_MODE_ALT_HOLD = 2,
-    SOL_MAVLINK_MODE_ATTITUDE = 3,
+    SOL_MAVLINK_MODE_ALTITUDE = 3,
```

Comment:
Sorry to be that late, but the previous version was correct. It really is Attitude is the orientation of an aircraft or spacecraft, relative to the direction of travel.